core/src/main/resources/hudson/model/Job/requirePOST_ru.properties

```diff
@@ -0,0 +1,23 @@
+# The MIT License
+#
+# Copyright (c) 2004-, Kohsuke Kawaguchi, Sun Microsystems, Inc., and a number of other of contributors
```

Comment:
can I leave the header empty in that case? or just leave '# This file is under the MIT License by authors'?